rewriter/Initializer.cc

```diff
@@ -29,6 +29,30 @@ bool isCopyableType(const ast::ExpressionPtr &typeExpr) {
     return true;
 }
 
+// Not checking for being T.proc, it's expected to be checked
+void maybeRemoveBind(core::Context ctx, ast::Send *send) {
+    auto lastSend = send;
+    while (send != nullptr) {
+        if (send->fun == core::Names::bind()) {
+            // `bind` is the last send in the chain of sends.
+            // This is incorrect syntax which is covered by the resolver.
+            // Safe to ignore.
+            if (send == lastSend) {
+                return;
+            }
```

Comment:
Do you mind writing a test for this case?

Reply:
Got into a sort of trouble testing it. The test requires marking the same line with two `# error:` comments and I'm not sure how to deal with it. Here is the test code:

```ruby
class TProcBindInInitializerLastSend
  extend T::Sig

  sig {params(blk: T.proc.bind(String)).void }
    @blk = blk
  end
end

```

The line `sig {params(blk: T.proc.bind(String)).void }` produces two errors now:
- Malformed T.proc: You must specify a return type
- Using `bind` is not permitted here

One way to get around it is to change the code in the PR, but it looks so clean and bullet proof that I'm hesitating to :)